docs/90_partners.md

```diff
@@ -1,17 +1,36 @@
 # Project partners
 
-* Walther Blom, Dell Technologies, High Education & Research
-* Jaco van Dijk, Dell Technologies, Higher Education
-* Bob Dröge, Center for Information Technology, University of Groningen
-* Robbert Eggermont, Delft University of Technology
-* Kenneth Hoste, HPC-UGent
-* Geert Jan Laanstra, Electrical Engineering, Mathematics and Computer Science (EEMCS), University of Twente, Enschede
-* Caspar van Leeuwen, SURF
-* Koen Mulderij, Delft University of Technology
-* Alan O'Cais, Juelich Supercomputing Centre
-* Thomas Röblitz, University of Oslo
-* Mark Sharpley, University of Cambridge Research Computing Services Division
-* Peter Stol, Vrije Universiteit Amsterdam
-* Marco Verdicchio, SURF
-* Bas van der Vlies, SURF
-* Henk-Jan Zilverberg, Center for Information Technology, University of Groningen
+## Delft University of Technology
+* Robbert Eggermont
+* Koen Mulderij
+
+## Dell Technologies
+* Walther Blom, High Education & Research
+* Jaco van Dijk, Higher Education
+
+## HPC-UGent
```

Comment:
@mulderij Let's indeed go with Ghent University, maybe also mention the country?

```
## Ghent University (Belgium)

([https://www.ugent.be/hpc/en](https://www.ugent.be/hpc/en))

* Kenneth Hoste

Reply:
This would create a lot of white space, maybe we should go with the header as the link:

```suggestion
## [Ghent University (Belgium)](https://www.ugent.be/hpc/en)
```

Reply:
@mulderij works for me!

Reply:
@boegel @robberteggermont  I think my change request on "Ghent University " blocks merging, but it is outdated... What to do?

Reply:
When somebody suggests a change, you can go to the "Files changed" tab and accept the change, or comment and mark as resolved.

Reply:
(So resolve the change instead of updating the commit outside of the change request.)

Reply:
@mulderij See previous comment. You can probably also "Commit suggestion" (if you agree) and then "Resolve conversation".

Reply:
@robberteggermont Unfortunately I cannot commit the suggestion: It's greyed out since it's outdated.

Reply:
@mulderij When you mark this conversation as resolved (e.g. via the GitHub Pull Request page), you should be able to merge the pull request.